### PR TITLE
added isEdgeChromium

### PR DESCRIPTION
**Pull request by djhouseknecht**
Add `isEdgeChromium` check.

Check it out here: https://apps.inindca.com/browserama/add-edge-chromium/

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ if (!isNotBrowser) {
   isSafari = Object.prototype.toString.call(window.HTMLElement).indexOf('Constructor') > 0 || (window.safari && window.safari.pushNotification);
   isOpera = (!!window.opr && !!window.opr.addons) || !!window.opera;
   isEdge = !!window.StyleMedia;
+  isEdgeChromium = isChromeOrChromium && !!navigator.userAgent.match(/edg/i);
```
Comment (djhouseknecht) [inline index.js:24]: Used the same SO for this one: https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+@Library('pipeline-library') _
+
+webappPipeline {
+    slaveLabel = 'dev_v2'
+    nodeVersion = '10.16.2'
```
Comment (jensengar) [inline Jenkinsfile:5]: currently, node 14 is the active lts. node 10 is about to hit EOL so we should see if we can use 14.

File: package.json [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
   "name": "browserama",
-  "version": "3.1.1",
+  "version": "3.2.0",
   "description": "A browser detection package that uses feature detection instead of user agent",
   "main": "out/browserama.min.js",
   "files": [
     "index.js",
     "out"
   ],
+  "publishConfig": {
```
Comment (jensengar) [inline package.json:10]: technically i don't think this is needed but it's probably good to have. i think you only need this for private packages.

**Review by jensengar — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (djhouseknecht) [reply Jenkinsfile]: Good point. I switched to to node 14

**Pull request merged**